configuration/pih/scripts/visit/encounterTypeConfig.js

```diff
@@ -1142,6 +1142,30 @@ angular.module("encounterTypeConfig", [])
           }
         };
 
+        // outpatientConsult
+        encounterTypes["92fd09b4-5335-4f7e-9f63-b2a663fd09a6"] = {
+            DEFAULT: {
+                defaultState: "short",
+                shortTemplate: "templates/encounters/defaultEncounterShort.page",
+                longTemplate: "templates/encounters/defaultHtmlFormEncounterLong.page",
+                templateModelUrl: "/module/htmlformentry/encounter.json?encounter={{encounter.uuid}}",
+                showOnVisitList: false
+            },
+            'peru': {
+                defaultState: "short",
+                shortTemplate: "templates/encounters/defaultEncounterShort.page",
+                longTemplate: "templates/encounters/defaultEncounterShort.page",
+                icon: "fas fa-fw fa-gift",
+                editUrl: hfeStandardEditUrl,
+                showOnVisitList: true,
+                sections: [
+                    primaryCareHistory,
+                    primaryCareExam,
+                    primaryCareDx,
+                    primaryCarePlan
+                ]
+            }
+        }
         // obgyn
```

Comment:
Uhmmm. i didn't upload this. It was a .properties file from the messageproperties folder. That .js  file was probably from an older PR.

Reply:
Ah okay, yeah it looks like you pushed to an old branch. Please use a fresh branch for each PR.

Reply:
Will do. Maybe you can close this after merging the transalations?

Reply:
There isn't a way to merge a partial PR. But we can update this one.

Reply:
Actually, really, the easiest way to do this will be just

```
git checkout master
git pull
git checkout -b peru-tx
tx pull
git add configuration/messageproperties/*
git commit -m "Update translations"
git push origin peru-tx
```

and then open a new PR. Guaranteed clean history.